Practice2/src/styles/main.css

```diff
@@ -0,0 +1,11 @@
+/* abstracts */
+@import url(./variables/main-var.css);
```

Comment:
It's not consistent between folders. Why do we import `main-var.css` directly here instead of `all.css` like other folders?

Reply:
Fixed